verify/main.go

```diff
@@ -38,6 +39,24 @@ type deps struct {
 	ctx    context.Context
 }
 
+/*
+HTTPClient interface that allows us perform mocking
+*/
+type HTTPClient interface {
+	Post(url string, contentType string, body io.Reader) (resp *http.Response, err error)
+}
+
+/*
+Client to use inplace of http.Client
+*/
+var (
+	Client HTTPClient
```

Comment:
Can we name it httpClient as we have client of firestore named as client?

Reply:
This is removed now